generator/internal/stub_rest_generator.cc

```diff
@@ -390,8 +417,9 @@ Default$stub_rest_class_name$::Async$method_name$(
 
   if (HasLongrunningMethod()) {
     // long running operation support methods
-    CcPrint(
-        R"""(
+    if (HasGRPCLongrunningOperation()) {
```

Comment:
I see that these are very close. Standardizing it could be a clean up for now / the future.

"google::longrunning::GetOperationRequest" vs. "$longrunning_get_operation_request_type$"
"absl::StrCat("/v1/", request.name())" vs. "$longrunning_get_operation_path$"